aspnetcore/fundamentals/configuration/options.md

```diff
@@ -38,29 +44,68 @@ Add a package reference to the [Microsoft.Extensions.Options.ConfigurationExtens
 
 ::: moniker-end
 
-## Basic options configuration
+## Options interfaces
+
+<xref:Microsoft.Extensions.Options.IOptionsMonitor`1> is used to retrieve options and manage options notifications for `TOptions` instances. <xref:Microsoft.Extensions.Options.IOptionsMonitor`1> supports the following scenarios:
+
+::: moniker range=">= aspnetcore-2.0"
+
+* Change notifications
+* [Named options](#named-options-support-with-iconfigurenamedoptions)
+* [Reloadable configuration](#reload-configuration-data-with-ioptionssnapshot)
+* Selective options invalidation (<xref:Microsoft.Extensions.Options.IOptionsMonitorCache`1>)
+* [Post-configuration](#options-post-configuration)
+
+<xref:Microsoft.Extensions.Options.IOptionsFactory`1> is responsible for creating new options instances. It has a single <xref:Microsoft.Extensions.Options.IOptionsFactory`1.Create*> method. The default implementation takes all registered <xref:Microsoft.Extensions.Options.IConfigureOptions`1> and <xref:Microsoft.Extensions.Options.IPostConfigureOptions`1> and runs all the configurations first, followed by the post-configuration. It distinguishes between <xref:Microsoft.Extensions.Options.IConfigureNamedOptions`1> and <xref:Microsoft.Extensions.Options.IConfigureOptions`1> and only calls the appropriate interface.
+
+<xref:Microsoft.Extensions.Options.IOptionsMonitorCache`1> is used by <xref:Microsoft.Extensions.Options.IOptionsMonitor`1> to cache `TOptions` instances. The <xref:Microsoft.Extensions.Options.IOptionsMonitorCache`1> invalidates options instances in the monitor so that the value is recomputed (<xref:Microsoft.Extensions.Options.IOptionsMonitorCache`1.TryRemove*>). Values can be manually introduced with <xref:Microsoft.Extensions.Options.IOptionsMonitorCache`1.TryAdd*>. The <xref:Microsoft.Extensions.Options.IOptionsMonitorCache`1.Clear*> method is used when all named instances should be recreated on demand.
+
+::: moniker-end
+
+::: moniker range="= aspnetcore-1.1"
+
+* Change notifications
+* [Reloadable configuration](#reload-configuration-data-with-ioptionssnapshot)
+
+::: moniker-end
+
+::: moniker range="< aspnetcore-1.1"
+
+* Change notifications
+
```

Comment:
This should also mention Named options maybe?

Reply:
And maybe also something to the effect of granular/selective cache invalidation via the IOptionsMonitorCache

Reply:
Never mind this is 1.0/1.x I see now